content/en/docs/ops/ambient/install/platform-prerequisites/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Platform-Specific Prerequisites
+description: Platform-Specific Prerequisites for Installing Istio Ambient Mode.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+This document covers any platform or environment specific prerequisites for installing Istio in ambient mode.
+
+## Platform
+
+### Minikube
+
+1. If you are using [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) with the [docker driver](https://minikube.sigs.k8s.io/docs/drivers/docker/), you may need to append `--set cni.cniNetnsDir="/var/run/docker/netns"` to the `helm install` command so that the Istio `cni-agent`node agent can correctly manage and capture pods on the node.
```

Comment:
Split long lines, and add a space between cni-agent and node.

```suggestion
1. If you are using [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) with the [docker driver](https://minikube.sigs.k8s.io/docs/drivers/docker/),
you may need to append `--set cni.cniNetnsDir="/var/run/docker/netns"` to the `helm install` command so that the Istio `cni-agent` node agent can correctly manage
and capture pods on the node.
```

Reply:
derp